design/ceph/topology-for-external-mode.md

```diff
@@ -0,0 +1,203 @@
+# Topology provisioning for external mode
```

Comment:
This is more like a user doc than a design doc. Shall we move it under `/Documentation`?